xla/pjrt/c/pjrt_c_api.h

```diff
@@ -1576,6 +1576,8 @@ struct PJRT_Executable_DeserializeAndLoad_Args {
   PJRT_Client* client;
   const char* serialized_executable;
   size_t serialized_executable_size;
+  const char* compile_options;
+  size_t compile_options_size;
```

Comment:
Also, due to ABI compatibility, I think these two arguments have to go at the end of the struct. This part is fuzzy to me due to the last arg here being the output.

@skye - Can you confirm here please?

Reply:
Reordered.